web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
```

Comment:
I don't think / should go to graphs, it'll scare off new users due to basically being a textbox with no context.

/graph is fundamentally a prometheus-debugging page, it doesn't make a good root page.